### PR TITLE
Social media

### LINKED ISSUE #435: Rich representation of content on social media
Jenn is looking into ways of making our content shine on social media platforms.
- Open Graph Protocol: http://ogp.me/
- Twitter cards: https://dev.twitter.com/cards/overview
~~- Pinterest: https://developers.pinterest.com/docs/rich-pins/articles/?~~ supported by schema.org

The current WC site uses both og and twitter markup, but this could be enhanced for specific media types.

### PR DESCRIPTION
**Pull request by gestchild**
## What is this PR trying to achieve?

Fixes #435

Although, not using twitter:image:alt or og:video as the content is not available from wordpress

## What does it look like?

Twitter:
twitter

Facebook:
facebook

Meta:
meta

### CODE REVIEW

**Comment by alexwlchan** [comment]
Why not a Summary Card with Large Image for Twitter, to match the Facebook version?

**Comment by gestchild** [comment]
Yep, could do.

screen shot 2017-02-08 at 14 38 02

@jennpb do you have a preference?

**Comment by jennpb** [comment]
@gestchild Do you know if it's either one or other? @RussellDornan as social media maestro, do you have a pref for small or large twittercard?

**Comment by gestchild** [comment]
Yep, either one or the other we have to specify

**Comment by RussellDornan** [comment]
Good day to you @gestchild and @jennpb  and @alexwlchan!

In the words of Nelly Furtdao: "The bigger the better, the bigger the better, the bigger the better..."

Large card please ❤️

**Comment by jennpb** [comment]
OK, large it is. @gestchild let's do as the man sez.

**Comment by gestchild** [comment]
@RussellDornan large card it is

File: server/model/article.js [hunk 1]
```diff
@@ -2,12 +2,14 @@
 import entities from 'entities';
 import {type Person} from './person';
 import {type Picture} from './picture';
+import {type Type} from './type';
```
Comment (jamesgorrie) [inline server/model/article.js]: `contentType` perhaps?

File: server/model/organization.js [hunk 2]
```diff
@@ -14,6 +15,7 @@ export const wellcomeCollection: Organization = {
   name: 'Wellcome Collection',
   url: 'https://wellcomecollection.org',
   logo: 'https://wellcomecollection.org/client/icons/apple-touch-icon.png',
+  twitterHandle: '@ExploreWellcome',
```
Comment (jamesgorrie) [inline server/model/organization.js]: It'd be good to figure out how we infer `sameAs` from `twitterHandle`, but, personally, I've no idea yet...

File: server/controllers/index.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
 // TODO: FlowType this module
 import {type Promo} from '../model/promo';
+import {wellcomeCollection} from '../model/organization';
 import {createPageConfig} from '../model/page-config';
 import {getPosts, getArticle} from '../services/wordpress';
 
+export const site = wellcomeCollection;
```
Comment (jamesgorrie) [inline server/controllers/index.js]: [We already add `organization` to the pageConfig](https://github.com/wellcometrust/wellcomecollection.org/blob/master/server/model/page-config.js#L16)

File: server/model/article.js (hunk 1 above)
Comment (gestchild) [inline server/model/article.js]: thanks, I didn't like this when I wrote it. Will change

File: server/controllers/index.js (hunk 3 above)
Comment (gestchild) [inline server/controllers/index.js]: oh yeah, I'll get rid of this

File: server/model/article.js [hunk 4]
```diff
@@ -2,12 +2,14 @@
 import entities from 'entities';
 import {type Person} from './person';
 import {type Picture} from './picture';
+import {type contentType} from './content-type';
```
Comment (jamesgorrie) [inline server/model/article.js]: Capital C please?

**Pull request merged**